client/src/components/ImageCard.vue

```diff
@@ -1,19 +1,17 @@
 <template>
   <div class='card'>
     <span class='asset'>
-      <img :src='require(`@/assets/media/${content.image ?? defaultImage}`)'
+      <img :src='require(`@/assets/media/${image}`)'
            alt=''>
     </span>
-    <h1>{{ content.header }}</h1>
-    <h2>{{ content.subHeader }}</h2>
+    <h1>{{ header }}</h1>
```

Comment:
Actually now that I've looked at this whole PR, since we have lg and xl of h1 headers maybe we should stick to our divs. What do you think?

Reply:
Looking at copper it looks like they have h1, h2, h3 which are 32px, 24px (bold) and 24px (no bold) respectively. I added the large and xl headers to match the mocks so they include 40px and 56px. If we go with copper I assume we would just use their largest (32px) which would differ pretty dramatically for some of the elements we have now. 

Alternatively we could discuss with Jackson and see if he has an opinion. As a short term solution I'd probably prefer just leaving our divs and reconciling with copper at a later date since this would either require changing the UI or making changes in copper.

Reply:
+1 let's leave the divs. Can you change this to one of the div classes so everything is the same

Reply:
Yeah totally missed this.